packages/happo-target-react-native/scripts/build-runner-android.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Building HappoRunner.apk"
+node ./node_modules/react-native/local-cli/cli.js run-android --root ./runner
```

Comment:
i did it for the ios one mainly because i was doing some `cd foo` and `cd -` in it... which is uglier in a package.json script. I don't like making package.json scripts too complicated as they get hard to parse real quick